tests/integration/Formats/SymphonyRtcFormatTests.cs

```diff
@@ -0,0 +1,77 @@
+using System.Threading.Tasks;
+using LogJoint.Postprocessing;
+using System.Linq;
+using System;
+using NFluent;
+
+namespace LogJoint.Tests.Integration.Symphony
+{
+	[IntegrationTestFixture]
+	class SymphonyRtcLogTests
+	{
+		[IntegrationTest]
+		public async Task LoadsLogFileAndEnablesPostprocessors(IContext context)
+		{
+			await context.Utils.EmulateFileDragAndDrop(await context.Samples.GetSampleAsLocalFile("rtcLog2018-09-15T21_41_44.195Z.txt"));
+
+			await context.Utils.WaitFor(() => context.Presentation.LoadedMessagesLogViewer.VisibleLines.Count > 0);
+			Check.That(context.Presentation.LoadedMessagesLogViewer.VisibleLines[0].Value).IsEqualTo(
+				"2018-09-15T21:41:34.133Z | TRACE(4) | stats-1 | RTCMediaStreamTrack_receiver_21.totalSamplesDuration=0");
+
+			var postprocessorsControls = context.Presentation.Postprocessing.SummaryView;
+			Check.That(postprocessorsControls.Timeline.Enabled).IsTrue();
+			Check.That(postprocessorsControls.SequenceDiagram.Enabled).IsTrue();
+			Check.That(postprocessorsControls.StateInspector.Enabled).IsTrue();
+			Check.That(postprocessorsControls.TimeSeries.Enabled).IsTrue();
+		}
+
+		[IntegrationTest]
+		public async Task CanRunStateInspectorPostprocessor(IContext context)
+		{
+			await context.Utils.EmulateFileDragAndDrop(await context.Samples.GetSampleAsLocalFile("rtcLog2018-09-15T21_41_44.195Z.txt"));
+
+			var postprocessorsControls = context.Presentation.Postprocessing.SummaryView;
+			await context.Utils.WaitFor(() => postprocessorsControls.StateInspector.Run != null);
+
+			postprocessorsControls.StateInspector.Run();
+
+			await context.Utils.WaitFor(() => postprocessorsControls.StateInspector.Show != null);
+
+			var symRtc = await context.Presentation.Postprocessing.StateInspector.Roots.FirstOrDefault(n => n.Id == "Symphony RTC");
+			Check.That(symRtc).IsNotNull();
+			var meeting1 = await symRtc.Children.FirstOrDefault(n => n.Id == "meeting-1");
+			Check.That(meeting1).IsNotNull();
+			var session1 = await meeting1.Children.FirstOrDefault(n => n.Id == "session-1");
+			Check.That(session1).IsNotNull();
+			var sessionIdChange = session1.ChangeHistory.FirstOrDefault(i => i.PropertyName == "session id");
+			Check.That(sessionIdChange).IsNotNull();
+			Check.That(sessionIdChange.Value).IsEqualTo("6357b48c-89c4-45d1-80c0-0a98245f7d6c");
+		}
+
+		[IntegrationTest]
+		public async Task CanCorrelateClientLogsFromChromedebugLogAndCSLogs(IContext context)
```

Comment:
I did not demo that. But when one downloads CS logs, the visualizers can look funny because of time skew between client log's time and CS's. In chromedebug log the time is local, so the diff UTC-vs-local is almost certainly present. The "fix clocks skew" postprocessor fixes that. This test verifies that flow.